### PR TITLE
Update Dicom json converter to handle Infinity values

### PR DESCRIPTION
**Pull request by arunmk-ms**
Fixes # [Issue 1725](https://github.com/fo-dicom/fo-dicom/issues/1725).

#### Checklist
- [X] The pull request branch is in sync with latest commit on the *fo-dicom/development* branch
- [ ] I have updated API documentation
- [X] I have included unit tests
- [X] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file

#### Changes proposed in this pull request:
- Update Dicom json converter to handle Infinity values
- Added unit tests for FL and FD VRs that can have Infinity and -Infinity values

### CODE REVIEW

**Comment by arunmk-ms** [comment]
Hi @gofal, please take a look at your convenience. Thanks

File: FO-DICOM.Core/Serialization/JsonDicomConverter.cs [hunk 1]
```diff
@@ -616,6 +616,14 @@ private static void WriteJsonElement<T>(Utf8JsonWriter writer, DicomElement elem
                     {
```
Comment (mrbean-bremen) [inline FO-DICOM.Core/Serialization/JsonDicomConverter.cs:616]: Looks like the check for `double.IsNan()` is also missing - can you add it while at it?

File: FO-DICOM.Core/Serialization/JsonDicomConverter.cs [hunk 2]
```diff
@@ -903,6 +911,28 @@ private static T[] ReadJsonMultiNumberOrStringValue<T>(ref Utf8JsonReader reader
                 {
                     childValues.Add((T)(float.NaN as object));
```
Comment (mrbean-bremen) [inline FO-DICOM.Core/Serialization/JsonDicomConverter.cs]: same here, of course

File: FO-DICOM.Core/Serialization/JsonDicomConverter.cs [hunk 3]
```diff
@@ -958,6 +988,28 @@ private static T[] ReadJsonMultiNumberValue<T>(ref Utf8JsonReader reader, GetVal
...
```
Comment (mrbean-bremen) [inline FO-DICOM.Core/Serialization/JsonDicomConverter.cs]: and here

**Review by mrbean-bremen — COMMENTED** [review]
Looks good, thanks! Do you want to add yourself to the contributors list?

File: FO-DICOM.Core/Serialization/JsonDicomConverter.cs (hunk 1 above)
Comment (arunmk-ms) [reply FO-DICOM.Core/Serialization/JsonDicomConverter.cs:616]: done

File: FO-DICOM.Core/Serialization/JsonDicomConverter.cs (hunk 2 above)
Comment (arunmk-ms) [reply FO-DICOM.Core/Serialization/JsonDicomConverter.cs]: done

File: FO-DICOM.Core/Serialization/JsonDicomConverter.cs (hunk 3 above)
Comment (arunmk-ms) [reply FO-DICOM.Core/Serialization/JsonDicomConverter.cs]: done

**Review by mrbean-bremen — APPROVED** [review]
Thanks!

**Review by gofal — APPROVED** [review]
Great, thanks!

**Pull request merged**